### DISCUSSION TITLE
Pimcore Logs not working

### DISCUSSION THREAD

**Discussion by Karthi96** [pimcore/pimcore#8102] [Q&A / Support] [answered]
Hi Team,

I'm running a pimcore 6.4.8 in local. I'm not able to see the pimcore logs on var/logs.prod.log. In code, I'm making an entry to logger::info("Test log message") but it didn't log.

How to get print the logs in the file ? Any idea ?

**Comment by dvesh3** [comment]
If you are using Symfony 4 then check path /var/**log/**{environment}.log. and there's no 6.4.8 version of Pimcore :)

**Reply by Karthi96** [reply]
sorry, typo error mine is 6.8.4, I can see the prod.log but it's not giving the logs of pimcore::info or error or critical or whatever? why so?

**Reply by dvesh3** [reply]
works for me with error/critical level 
![image](https://user-images.githubusercontent.com/5137917/107202507-90fa5f00-69fa-11eb-8c55-8083fc9902fd.png)

please keep in mind that log level set to "error" on prod.yml https://github.com/pimcore/pimcore/blob/6.8/bundles/CoreBundle/Resources/config/pimcore/prod.yml

**Reply by Karthi96** [reply]
Thanks, is there a way to make info logs on prod?

**Reply by dvesh3** [reply]
yes, redefine logging config from prod.yml in your config.yml with "level: info".

**Comment by Karthi96** [comment, marked as answer]
@dvesh3 Thanks. As you said logger level can be controlled here and yes this is my issue and now answered.
